velox/expression/VectorUdfTypeSystem.h

```diff
@@ -602,12 +602,12 @@ struct VectorWriter<ArrayProxyT<V>> {
   }
 
   void reset() {
-    proxy_.valuesOffset_ = 0;
+    elementWriter_.valuesOffset_ = 0;
   }
 
   vector_t* arrayVector_ = nullptr;
 
-  exec_out_t proxy_;
+  exec_out_t elementWriter_;
```

Comment:
would arrayWriter_ be a better name for this?

elementWriter_ sounds like it's writing the elements of the Array, which is also what childWriter_ sounds like.

since elementWriter_ is actually the ArrayProxy (which gets renamed to ArrayWriter in a downstream change) arrayWriter_ might be more clear

Reply:
I did this change in the a following diff that focus on replacing proxy_ naming with elementWriter_